ocpmodels/trainers/forces_trainer.py

```diff
@@ -593,13 +602,14 @@ def run_relaxations(self, split="val", epoch=None):
             )
 
             if self.config["task"].get("write_pos", False):
-                ids = relaxed_batch.id
+                systemids = [str(i) for i in relaxed_batch.sid.tolist()]
                 natoms = relaxed_batch.natoms.tolist()
                 positions = torch.split(relaxed_batch.pos, natoms)
                 batch_relaxed_positions = [pos.tolist() for pos in positions]
-                relaxed_positions += list(
-                    zip(ids.tolist(), batch_relaxed_positions)
-                )
+                relaxed_positions += [
+                    {"ids": i, "positions": j}
```

Comment:
This should probably be `id` instead of `ids`, right?

Reply:
oops typo, yup.